### ISSUE TITLE
Support reading EBU-TT-D documents

### ISSUE THREAD

**Issue by nigelmegitt** [bbc/ebu-tt-live-toolkit#62]
Note gotcha: `EBUTTDDocument.create_from_xml` has a nasty hack workaround that tweaks the name of the document element to `ttd` instead of `tt`. Found during testing, a bug in `ebu_tt_live/documents/ebuttd.py` - it doesn't check for the presence of the prefix, so only works when the default namespace is the TTML namespace and no prefix is used on the `tt` element, i.e. fails for `tt:tt`

Fix is:
```python
        if xml_dom.documentElement.tagName == xml_dom.documentElement.prefix + ':tt':
            xml_dom.documentElement.tagName = xml_dom.documentElement.prefix +  ':ttd'
```

In other words, test for the prefix.
